apps/projects/templates/a4_candy_projects/includes/project_module_tile.html

```diff
@@ -5,7 +5,7 @@
     <a href="{{ module.get_absolute_url }}">
         <div class="participation-tile__type">
             <div class="participation-tile__content">
-                <h3 class="participation-tile__title">{{ module.name|truncatechars:100 }}</h3>
+                <h4 class="participation-tile__title">{{ module.name|truncatechars:100 }}</h4>
```

Comment:
We cannot use the headlines for styling or font sizes as this will break a11y. Does this have the appropriate order of an h1, h2 and h3 before? Otherwise, you could add an utility class `u-h4-font-size` or something like that, that has the same sizing behaviour as an h4, but still use the h3 for the structure.

Reply:
yes true. I restored it to h3 again and used the class, which is only used here to make the font-size change. That way I avoid to introduce a couple of utility classes (because of media queries).

Note: Anyway h2 is missing in the structure, but maybe because it is my dev-testing-content which does not contain proper descriptions etc.